lock-api/src/main/java/com/palantir/lock/client/UnreliableTimeLockService.java

```diff
@@ -41,20 +41,25 @@
  * acquiring. This is useful for testing the behavior of clients when locks are lost.
  */
 public final class UnreliableTimeLockService implements TimelockService {
```

Comment:
JavaDoc is a bit outdated now. Let's include new behavior or change the doc to say something more generic than it already says.